sdk/monitor/azure-monitor-opentelemetry/samples/tracing/django/sample/sample/settings.py

```diff
@@ -29,7 +30,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
```

Comment:
Can we just exempt the samples folder as a whole in mypy.ini?

Reply:
Samples are included as part of the workflow. Seems like most other azure services are enabling type checking for samples.